src/copy.jl

```diff
@@ -52,7 +52,7 @@ function execute(
         if parameters === nothing
             result = _execute(jl_conn.conn, copy.query)
         else
-            result = _execute(jl_conn.conn, copy.query, pointer_params)
+            GC.@preserve string_params result = _execute(jl_conn.conn, copy.query, pointer_params)
```

Comment:
```suggestion
            result = GC.@preserve string_params _execute(jl_conn.conn, copy.query, pointer_params)
```
looks cleaner imho

Reply:
Done in https://github.com/iamed2/LibPQ.jl/pull/276/commits/1d60f24e422f63d57ef9d4dcef7177cc2adcc00c